main.cpp

```diff
@@ -74,11 +24,15 @@ void testPairToNumber(
 }
 
 int main() {
+
     testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
     testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
 
     testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
     testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
 
+    std::cout << "\nReference Manual:\n";
+    TelCoColorCoder::PrintReferenceManual();
```

Comment:
Like in the other `test...` functions, can you design asserts for `PrintReferenceManual`, to reduce the manual test burden? Otherwise, it requires manual inspection of 25 lines of output